go.mod

```diff
@@ -7,21 +7,27 @@ require (
 	cloud.google.com/go/pubsub v1.3.1
 	cloud.google.com/go/storage v1.6.0
 	github.com/crossplane/crossplane v0.13.0-rc.0.20200924053741-18324aa15a19
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20200924144923-240dbf0821e6
-	github.com/crossplane/crossplane-tools v0.0.0-20200923030414-95b434323cd4
+	github.com/crossplane/crossplane-runtime v0.9.1-0.20201001234308-3f1afd106a8c
+	github.com/crossplane/crossplane-tools v0.0.0-20201001224552-fb258cc0eb30
+	github.com/go-openapi/spec v0.19.5 // indirect
 	github.com/google/go-cmp v0.4.0
 	github.com/googleapis/gax-go v1.0.3
 	github.com/googleapis/gax-go/v2 v2.0.5
 	github.com/imdario/mergo v0.3.9
+	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mitchellh/copystructure v1.0.0
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.0.0 // indirect
+	github.com/stretchr/testify v1.5.1 // indirect
+	golang.org/x/crypto v0.0.0-20200414173820-0848c9571904 // indirect
 	google.golang.org/api v0.21.0
 	google.golang.org/genproto v0.0.0-20200331122359-1ee6d9798940
 	google.golang.org/grpc v1.28.0
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
+	gopkg.in/yaml.v3 v3.0.0-20191120175047-4206685974f2 // indirect
+	k8s.io/api v0.18.8
+	k8s.io/apimachinery v0.18.8
+	k8s.io/client-go v0.18.8
 	sigs.k8s.io/controller-runtime v0.6.2
-	sigs.k8s.io/controller-tools v0.2.4
+	sigs.k8s.io/controller-tools v0.3.0
```

Comment:
Are these changes expected? I didn't notice them in the other PRs and they don't look they are related to the changes here

Reply:
No, these snuck in. Fixed.

Reply:
Hmm a number of these are still showing up for me 🤔

Reply:
You're right - fixed for really reals now.